docs/dashboards/linked-filters.md

```diff
@@ -0,0 +1,70 @@
+
+- In the same table;
+- In two different tables that have a foreign key relationship specified in [table metadata in Admin settings](../data-modeling/metadata-editing.md)
+- In two different tables that have a foreign key relationships to one or more intermediate tables specified in [table metadata in Admin settings](../data-modeling/metadata-editing.md)
+
+![Setting up foreign key in table metadata](./images/foreign-key-linked-filters.png)
+
+Note that Metabase will not show an error if you try to set up linked filters with improperly linked columns, but you'll see that the values in child linked filter will not be filtered by parent filter. See [Troubleshooting link filters](../troubleshooting-guide/linked-filters.md) for more troubleshooting tips.
+
+## Set up linked filters
+
+You can link a child filter to one or more parent filters. The child filter (filter whose values are determined by another filter), must be an ID, Location, or Text or Category filter. Parent filters can have any [filter type](./filters.md).
+
+To link a child filter on a dashboard to one or more parent filters:
+
+1. Edit the dashboard by clicking on the pencil icon in the top right of the dashboard;
+2. Edit the child filter by clicking on the gear icon in the filter;
+3. In the filter settings sidebar on the right, switch to **Linked filters** tab;
+4. Select the parent filters.
+
+![Linked filters](./images/linked-filter.png)
+
+The filters you select in the **linked filters** tab will be used to limit the choices of filter values for the child filters.
+
+## Linked filters limitations
+
+### Linked filters only use table-level information
+
+A single dashboard filter can be wired to the same column on multiple dashboard cards, so dashboard filters — including linked filters — can only use table-level information about the column to populate possible filter values, and can't rely on any logic included in any specific dashboard card.
+
+For linked filters specifically, this means that they can only use relationships specified on the table level in Table Metadata, and they will ignore any relationships specified only the model or question level.
+
+In particular, this means that:
+
+- Joining tables in a question or a model source, or even setting foreign key relationships in model metadata only, is **not sufficient** to enable linked filter values. You need to specify relationships in table metadata.
+
+- Linked dashboard filters will not use any filter or join logic from any underlying card or model.
+
+  For example, if you have State and City columns, and you build a model that filters out any data with `City = San Francisco`, then build a question based on this model and add it to a dashboard with linked State and City filters, selecting `State = CA` will still show San Francisco as an option for the City filter (although of course actually selecting this option will return no data for the question built on the "no San Francisco" model) .
```

Comment:
```suggestion
For example, say you have a table with State and City columns, and you build a model that filters out rows with `City = San Francisco`. You ask a question based on that model, and add it to a dashboard. You add State and City filters to the dashboard, and link them. If you select were to select `State = CA`, the city filter may still show `San Francisco` as option, even though there are no records with `San Francisco` in the question and the underlying model, because the filter only "knows" about the underlying table metadata (which includes sample values for the column).
```

Does this clarify what happens and why?